native/core/src/parquet/parquet_support.rs

```diff
@@ -1861,6 +1864,40 @@ fn trim_end(s: &str) -> &str {
     }
 }
 
+#[cfg(not(feature = "hdfs"))]
+pub(crate) fn register_object_store(
+    session_context: Arc<SessionContext>,
+) -> Result<(), ExecutionError> {
+    let object_store = object_store::local::LocalFileSystem::new();
```

Comment:
It doesn't have to be only a local file system.

Reply:
It depends on the feature enabled for the Comet. LocalFileSystem is by default if no specific features selected. 
the annotation on this method is 
``` 
#[cfg(not(feature = "hdfs"))]
```
This allows to plugin other features like S3, etc